### PR TITLE
Introduce 0-jumphost step before bootstrap

### PR DESCRIPTION
**Pull request by lavjain**

### CODE REVIEW

File: solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+sudo snap install docker && sudo snap start docker
+
+# Install Kustomize
+export KUSTOMIZE_VERSION=4.5.7
+wget https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv"$KUSTOMIZE_VERSION"/kustomize_v"$KUSTOMIZE_VERSION"_linux_amd64.tar.gz
+tar -xzf ./kustomize_v"$KUSTOMIZE_VERSION"_linux_amd64.tar.gz
+sudo cp kustomize /usr/local/bin/
+
+# Install Skaffold
+export SKAFFOLD_VERSION=1.39.2
```
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh]: let's set the version to v2.4.0 for skaffold

File: solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# Update system packages
+sudo apt-get update -y && sudo apt-get autoremove -y
+sudo DEBIAN_FRONTEND=noninteractive apt-get upgrade -yq
+sudo apt-get install apt-transport-https python3-pip pipenv redis-tools unzip -y
+
+sudo addgroup --system docker
+sudo snap install docker && sudo snap start docker
+
+# Install Kustomize
+export KUSTOMIZE_VERSION=4.5.7
```
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh]: Set kustomize to 5.0 to align with the readme.

File: solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# Install terraform, helm, kubectl and other tools
+sudo snap install terraform --classic
+sudo snap install helm --classic
+sudo snap install kubectl --classic
+pushd /usr/bin
+sudo ln -s python3 python
+popd
+python -m pip install --upgrade pip
+python -m pip install google-cloud-firestore google-cloud-bigquery firestore-admin
+touch completed
```
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh]: can we rename the file to `jumphost_ready`?

**Review by jonchenn — CHANGES_REQUESTED** [review]

File: solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+# Install terraform, helm, kubectl and other tools
+sudo snap install terraform --classic
+sudo snap install helm --classic
+sudo snap install kubectl --classic
+pushd /usr/bin
+sudo ln -s python3 python
+popd
+python -m pip install --upgrade pip
+python -m pip install google-cloud-firestore google-cloud-bigquery firestore-admin
```
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh]: Please install `pip install solutions-builder`

File: solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/scripts/bastion_startup.sh:1]: Please move this file to 0-jumphost

File: solutions_builder/template_root/terraform/stages/2-foundation/main.tf [hunk 5]
```diff
@@ -62,6 +62,10 @@ locals {
 # Used to retrieve project_number later
 data "google_project" "project" {}
 
+resource "google_identity_platform_config" "default" {
+  project = var.project_id
```
Comment (jonchenn) [inline solutions_builder/template_root/terraform/stages/2-foundation/main.tf]: why is this added in this PR?

**Review by jonchenn — CHANGES_REQUESTED** [review]

File: solutions_builder/template_root/terraform/stages/2-foundation/main.tf (hunk 5 above)
Comment (lavjain) [reply solutions_builder/template_root/terraform/stages/2-foundation/main.tf]: This was from the previous PR to enable identity platform. I will remove it

**Review by jonchenn — APPROVED** [review]
LGTM
